datamodel/src/main/java/com/hartwig/serve/datamodel/ActionableEvent.java

```diff
@@ -29,7 +29,10 @@ public interface ActionableEvent {
     Set<CancerType> blacklistCancerTypes();
 
     @NotNull
-    String description();
+    String efficacyDescription();
+
+    @NotNull
+    String efficacyDescriptionYear();
```

Comment:
This looks odd as a String. Shouldn't it a nullable integer?

Reply:
I agree on the Integer part, but shouldn't it remain @NotNull since we fall back on therapy and entry dates for this value?

Reply:
If it is always there it should be notnull indeed. Wasn't really sure when I wrote this comment.

One minor note, the type should just be `int` rather than `notnull integer`, since that's effectively what it is (a primitive type cannot be null so should be used instead of objects in case of notnull)